packages/story-editor/src/components/floatingMenu/elements/borderWidthAndColor.js

```diff
@@ -78,13 +78,13 @@ function BorderWidthAndColor() {
   );
   // We only allow editing the current border width, if all borders are identical
   const hasUniformBorder =
-    border.left === border.right &&
```

Comment:
Potentially, yes. I think the error here is basically in regards to `border` being null at this point, hence it blows up the editor with an error when it happens, but it could be a deeper core issue. This resolves, for now, the issue at hand with the error, but might be worth taking a deeper look at.